extensions/amp-analytics/0.1/events.js

```diff
@@ -361,7 +361,11 @@ export class ClickEventTracker extends EventTracker {
     /** @private {!Observable<!Event>} */
     this.clickObservable_ = new Observable();
 
-    /** @private @const */
+    /**
+     * @private {function(!Event)}
+     * @const
+     * @param {!Event} e
```

Comment:
@jridgewell I had to add this param to pass the linter. however, check-types seems to not like it. what's the right way to do here?